.github/workflows/pr-build.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         working-directory: ./air-quality-backend
 
       - name: Run unit tests
-        run: python -m pytest --cov=src/air_quality --cov=scripts --cov-report= tests
+        run: python -m pytest --cov=src/air_quality --cov-report= tests
```

Comment:
I'll make a separate ticket to unit test the scripts folder - it's a bit more tricky than it seems I think